source/Draft/Exceptions/ExceptionHandling.cs

```diff
@@ -36,6 +36,8 @@ public static EtcdException ProcessException(this FlurlHttpException This)
 
             var etcdError = This.GetResponseJson<EtcdError>();
 
+            if (etcdError == null) { return new UnknownErrorException(This.Message); }
```

Comment:
This is where the null reference exception was thrown originally, so I just handled it this way in case it ever comes up again in another circumstance.